src/protocols/bdx/TransferFacilitator.cpp

```diff
@@ -102,6 +102,7 @@ CHIP_ERROR Responder::PrepareForTransfer(System::Layer * layer, TransferRole rol
 
     mPollFreq    = pollFreq;
     mSystemLayer = layer;
+    ChipLogProgress(BDX, "%s: Start polling for messages", __FUNCTION__);
```

Comment:
Please do not use __FUNCTION__ in device code.  It leads to very bloaty code.

But also, shouldn't this log line be down where the StartTimer call is?  Because if `WaitForTransfer` errors out, we are not in fact polling....

Reply:
done.